easybuild/easyconfigs/r/RepastHPC/RepastHPC-2.2.0-foss-2016a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'RepastHPC'
+version = '2.2.0'
+
+homepage = 'https://repast.github.io/'
+description = """The Repast Suite is a family of advanced, free, and open source agent-based modeling and simulation
+platforms that have collectively been under continuous development for over 15 years: 
+Repast for High Performance Computing 2.2.0, released on 30 September 2016, is a lean and expert-focused C++-based 
+modeling system that is designed for use on large computing clusters and supercomputers."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'usempi': True}
+
+sources = ["repast_hpc-%(version)s.tgz"]
+source_urls = ['https://github.com/Repast/repast.hpc/releases/download/v%(version)s']
+
+dependencies = [
+    ('netCDF-C++', '4.2'),
+    ('Boost', '1.60.0'),
+]
+
+configopts = '--with-boost=$EBROOTBOOST --with-netcdf=$EBROOTNETCDFMINCPLUSPLUS'
+
+sanity_check_paths = {
+    'files': ['lib/librelogo-%(version)s.%s' % SHLIB_EXT, 'lib/librepast_hpc-%(version)s.%s' % SHLIB_EXT ],
```

Comment:
@mboisson You need to use `%%(version)s` here (i.e. escape the `%`), to ensure that you have `%(version)s` in the string after string templating with `% SHLIB_EXT` is done